TodoList/Extensions/UIViewExtensions.swift

```diff
@@ -2,6 +2,9 @@ import UIKit
 
 extension UIView {
     
+    /// Builds a nib.
```

Comment:
I'd do something like this to better understand how it builds the nib.
```suggestion
    /// Builds a nib using the object's type as nibName.
```